lib/urlwatch/reporters.py

```diff
@@ -383,6 +383,8 @@ class EMailReporter(TextReporter):
 
     def submit(self):
         filtered_job_states = list(self.report.get_filtered_job_states(self.job_states))
+        cfg = self.report.config['report']['email']
```

Comment:
You should be able to use `self.config` instead of `self.report.config['report']['email']` -- this is also more flexible, as it doesn't hardcode the reporter kind and/or path to the config.